firefox/package.json

```diff
@@ -6,7 +6,7 @@
 	"author": "honestbleeps",
 	"homepage": "http://redditenhancementsuite.com/",
 	"license": "GPL-3.0",
-	"version": "{{prop?version!../package.json}}",
+	"version": "{{to-string!./getVersion}}",
```

Comment:
FYI, JPM's requirement for Semver is documented here: https://developer.mozilla.org/en-US/Add-ons/SDK/Tools/package_json